### PR TITLE
Add workspace-client

### PR DESCRIPTION
**Pull request by olexii4**
This PR added workspace-client instead of the native implementation.

Signed-off-by: Oleksii Orel <oorel@redhat.com>

### CODE REVIEW

File: src/services/api/workspace.ts [hunk 1]
```diff
@@ -70,6 +71,7 @@ export async function createWorkspaceFromDevfile(
   const attrs = Object.keys(attributes).map(key => `${key}:${attributes[key]}`);
 
   try {
+    // TODO rework this solution with WorkspaceClient
```
Comment (sleshchenko) [inline src/services/api/workspace.ts]: It depends on the issue about updating Workspace Client, right? If so - please add it there https://github.com/eclipse/che/issues/17700

**Comment by akurinnoy** [comment]
Do we still need `src/services/api/workspace.ts`? Why don't you directly use restApiClient to create, update, delete workspaces etc.?

File: src/services/api/workspace.ts (hunk 1 above)
Comment (olexii4) [reply src/services/api/workspace.ts]: I have done

**Comment by olexii4** [comment]
@akurinnoy It depends on https://github.com/eclipse/che/issues/17700.

It is going to remove src/services/api/workspace.ts after.

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**